data/nlu/nlu.yml

```diff
@@ -5,8 +5,8 @@ nlu:
     - are you a bot?
     - am i speaking to an ai?
     - are you a human or a bot?
-    - are you real? 
-    - are you virtual? 
+    - are you real?
+    - are you virtual?
```

Comment:
Anonymous endpoints are not allowed at this time.

https://github.com/sabiocode/wiki/blob/master/general/code-review/Routing-Api.md#anonymous-endpoints